proto/spawn_diff.proto

```diff
@@ -12,7 +12,7 @@ message SpawnDiff {
   oneof diff {
     OldOnly old_only = 5;
     NewOnly new_only = 6;
-    Modified modified = 7;
+    Common common = 7;
```

Comment:
Nit: Unclear why this rename is needed.

Reply:
I found that `Modified` is not quite the right term as users don't directly modify a spawn, so I wanted to opt for something that fits better with `OldOnly` and `NewOnly`. 

It doesn't have to be part of this PR though, I changed the name back to make the diff easier to read.